keps/sig-ui/NNNN-kep-template/README.md

```diff
@@ -0,0 +1,810 @@
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This proposal aims to improve the graphical user interface experience in Kubernetes by introducing a Kubernetes Desktop application and an improved in-cluster user interface, both based on the Headlamp project.
+
+## Motivation
+
+The driving motivation behind this proposal is to provide both a great out of the box experience for end users and a foundation for vendors and other projects to integrate with and build upon. Essentially, we think Headlamps' ability to be adapted and used across many scenarios allows it to serve as a central hub for the CNCF community and projects.
+
+### Background
+
+Headlamp is a Kubernetes UI that can run as a Desktop application and an in-cluster web interface. It is designed to be extended using its robust plugin system. This allows vendors or projects to adapt it to their use case and for the project to deliver builds for various use cases.
+
+Headlamp is currently available in two versions.
+* **Headlamp** refers to the project and the default distribution of Headlamp which ships with core Kubernetes UI functionality (see the next bullet point) plus a set of plugins that the project thinks are generally useful for all users in order to provide a rich and relevant Kubernetes user experience.
+* **Headlamp Base** does not include any plugins. It only includes features that the project deems universally desired in a Kubernetes UI. This is designed for builders looking to have a solid base on which to extend the user experience. They would generally create custom plugins to customize the user experience, include any existing plugins they desire and then theme the UI to fit their branding.
+
+Each version is available as a desktop experience and an in-cluster experience. For the default build, the set of plugins differs between the in-cluster and desktop installations as there are different user profiles for each environment. For example, the plugin that allows managing plugins is not included in the in-cluster environment as the cluster admin generally determines which plugins to provide to the cluster users.
+
+### Existing Dashboard project
```

Comment:
I'd just remove this title and use it as motivation (removing the background section)